controllers/clusterconfig_controller.go

```diff
@@ -20,6 +20,8 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/openshift/cluster-relocation-service/internal/certs"
+	"k8s.io/apimachinery/pkg/api/errors"
```

Comment:
minor: since we have logical seperation  already could we move these deps out of core.